src/JobFeed/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { useQuery } from '@apollo/client';
+import {
+  Card, CardContent, CircularProgress, Typography, CardActions, Button, Link,
+} from '@material-ui/core';
+import { QUERY_JOBS } from '../graphql/queries';
+import { NOT_FOUND_PATH } from '../paths';
+
+function JobFeed() {
+  const { loading, error, data } = useQuery(QUERY_JOBS);
+  if (loading) {
+    return (
+      <CircularProgress />
+    );
+  }
+  if (error) {
```

Comment:
Please add a line break before if statements that return jsx to make easier to read